lightning/src/ln/channelmanager.rs

```diff
@@ -357,6 +357,8 @@ pub struct ChannelManager<ChanSigner: ChannelKeys, M: Deref> where M::Target: Ma
 	channel_state: Mutex<ChannelHolder<ChanSigner>>,
 	our_network_key: SecretKey,
 
+	last_node_announcement_serial: AtomicUsize,
```

Comment:
91f4e5d

Document field purpose "Used as a monotonic counter for node announcement timestamp"